src/odemis/gui/cont/tabs.py

```diff
@@ -508,20 +508,30 @@ def load_data(self, data):
         # Create streams from data
         streams = data_to_static_streams(data)
 
-        # TODO: Clear previous overview streams
-        self.clear_data()
-
         for s in streams:
+            self.clear_data(s)
             scont = self._overview_stream_controller.addStream(s, add_to_view=True)
             scont.stream_panel.show_remove_btn(True)
 
-    def clear_data(self):
+    def clear_data(self, new_stream=None):
         """
         Clear the tab data upon resetting the project:
         - Clear overview map streams
-        - Clear live streams data
-        """
-        self._overview_stream_controller.clear()
+        @:param new_stream: (StaticStream or None) the newly acquired overview stream
+        """
+        # Clear old static streams from the view
+        # TODO: Remove old streams from the panel as well, currently it's hidden
```

Comment:
Can you explain the issue more? I don't get what's still happening wrong.

Reply:
What's happening wrong is that the old image is removed from the view, but it still stays on the panel hidden (see attached image)
![Screenshot from 2021-04-09 15-34-57 png](https://user-images.githubusercontent.com/6759564/114189040-0aa2c080-994a-11eb-9d04-479572444036.png)